packages/components/src/horizontal-bar-list/stats-card.tsx

```diff
@@ -79,7 +79,11 @@ const StatsCard = ( props: StatsCardProps ) => {
 	// (main item, optional additional columns and value)
 	const splitHeaderNode = (
 		<div
-			className={ `${ BASE_CLASS_NAME }-header ${ headerClassName } ${ BASE_CLASS_NAME }-header--split` }
+			className={ clsx(
+				`${ BASE_CLASS_NAME }-header`,
+				headerClassName,
+				`${ BASE_CLASS_NAME }-header--split`
+			) }
```

Comment:
I noticed the `headerClassName` can be `undefined`, and that gets added as a class name; this change avoids adding undefined as a class name.

![Arc -2025-03-24 at 16 23 44@2x](https://github.com/user-attachments/assets/c13bef92-0fac-49da-ad7e-a2bcf133f42f)

Reply:
Nice catch! 👍🏼

Reply:
Good catch 👍